### DISCUSSION TITLE
【委托】求大佬教学

### DISCUSSION THREAD

**Discussion by Dongshan-panda** [share121/inter-knot#5888] [公告]
第一次玩米游，该怎么提升数值，又要怎么刷词条？
![17350157873536a687355519b2ea63372b475b81b9672373dc73eb2fcd5f69929fc6809e35b26 0](https://github.com/user-attachments/assets/2caa075b-2ddd-49c4-b222-290799cc45fb)

**Comment by fangtichang** [comment]
简单说一下我个人的看法

在绝区零里提升数值主要有三个方向，技能加点，天赋，以及驱动盘。（其实还有命座）

天赋挺好拉的，一般来说是全部加满。

技能的话考虑到耗费的资源比较多，点击角色的任意技能，它右下角有个培养建议，会告诉你优先提什么，次优先提什么，照着它的那个推荐点就行。

至于驱动盘的词条，因为每个角色所需要的词条属性是不一样的，这边建议：点击更多→绳匠工具，里面有每个角色的养成指南，有驱动盘推荐，以及对应角色所需要的词条，照着那上面的刷就行。

另外就是，他这个1，2，3号位的驱动盘由于主词条是固定的，只用刷副词条就OK，所以123号盘相对来说比较好刷，而456号盘就比较难了，不仅主词条要选对，而且副词条也不能太差，有时候刷个主词条都要费老大劲。

实在刷不出来相应主词条的话可以去音像店用调律校音器（一个比较珍稀的S级道具）定向一个。

以上是我个人的一些简单理解，希望对你有帮助，同时也欢迎其他大佬补充~

**Comment by zean396** [comment]
这个图片看一次笑一次😄
